packages/frontend/src/pages/MinimalDashboard.tsx

```diff
@@ -24,6 +32,12 @@ const MinimalDashboard: FC = () => {
         error,
     } = useDashboardQuery(dashboardUuid);
 
+    const dashboardFilters: DashboardFilters | undefined = useMemo(() => {
+        const searchParams = new URLSearchParams(window.location.search);
```

Comment:
Get the filters from the URL on minimal dashboard and send it to the chart instead of using the DashboardProvider